### DISCUSSION TITLE
Ember Waste Mystery Creature

### DISCUSSION THREAD

**Discussion by SultanRTG** [endless-sky/endless-sky#9909] [Q&A] [answered]
Is the storyline connected to the Ember Waste Node mystery creature finished, or is there more planned for it?

**Comment by Zitchas** [comment, marked as answer]
There is much more story planned for it. There will be a little more info in chapters 3 & 4 of the Remnant/Ember Waste campaign, but a full exposition won't be until quite a bit later.
